web/src/main/resources/com/graphhopper/http/README.MD

```diff
@@ -0,0 +1,5 @@
+The Polyongs in this folder are from here: https://github.com/datasets/geo-countries and provided under ODC Public Domain Dedication and Licence (PDDL)
+
+The Geojson has roughly 25MB, so I reduced it with http://www.mapshaper.org/ using Douglas Peuckar at 10%. The resulting file is 2.4mb.
+
+The rules are taken from the OSM Wiki: ttps://wiki.openstreetmap.org/wiki/OSM_tags_for_routing/Access-Restrictions
```

Comment:
Should be in the NOTICE.md instead (?)

And why is this and countries.geo.json in package com/graphhopper/http ?